src/Utility/Security.php

```diff
@@ -48,16 +48,6 @@ class Security
     protected static $_instance;
 
     /**
-     * Generate authorization hash.
-     *
-     * @return string Hash
-     */
-    public static function generateAuthKey()
```

Comment:
This is a breaking change at this point. We forgot to deprecate this during the 3.0 development :cry: